index.html

```diff
@@ -2029,9 +2029,16 @@ <h2>Verification Relationships</h2>
 <code><a>keyAgreement</a></code> property needs to be used for that.
       </p>
       <p>
-This specification defines several <a>verification relationships</a> below. A
-<a>DID document</a> MAY include any of these, or other properties, to express
-a specific <a>verification relationship</a>. In order to maximize global
+The <a>DID document</a> does not express revoked keys using a <a>verification
```

Comment:
While I would prefer this to be true, I am fairly certain others don't agree. @oed @dhh1128 ... I suppose you would agree with `the latest <a>DID document</a>  does not express revoked keys`... which would introduce the concept of time / when revocation happened... I fear the complexity of such language and the security issues it produces / attempts to resolve.

I think the sentence cannot be true, without defining revocation...  a DID Document might contain revoked GPG keys OR keys that have had their private component tweeted, and where the attacker controls the did document... 

this is bad, it means that the attacker gains the value of all keys in a did document and can use this statement to argue that since they are present they are authoritative... "look I have your house keys, I must own your house"...

are we saying it's impossible for a did document to contain "revoked keys" isn't that like saying "DID Documents" do not contain emojis ?

this section also contradicts the informative section on "verification method revocation".... 

This is a topic it would have been better to spend call time on than `@context`.

Reply:
Yeah, this only makes sense if it refer to the *latest* DID document. If you resolve a DID document with the `versionId` param you should expect to get the state of the document at that point in time!

Reply:
+1 to clarifying with "latest" language, that makes the statement more truthy.

> this is bad, it means that if we keep the propose change, when the attacker gains the value of all / some keys in a did document the attacker can use this statement to argue that since the attacker has not revoked the keys and they are present they are authoritative.

It's not good or bad, it's just reality. We can weaken the language, but the contract here is -- if the key is in the DID Document (and you have no other signals to say the key has been compromised), then you should assume the key hasn't been compromised.

That is true for any system that depends on PKI.

> are we saying it's impossible for a did document to contain "revoked keys" 

No, we are not saying that. I'll try to put in language to make sure people don't get that impression (although, I'd argue that's a pretty strange thing to take away from a read of the specification given the security/privacy considerations sections). We might want to write more about this in security considerations.

Reply:
> Even if a verification method is present in a DID Document, additional information, such as a public key revocation certificate, or an external allow or deny list, might be used to determine whether a verification method has been revoked.

https://w3c.github.io/did-core/#verification-method-revocation

perhaps this inconsistency is no longer present once `latest` is added :)